[14.0][ADD] Add stock_split_picking_kit (#1829)
OCA/stock-logistics-workflow: stock_split_picking_kit/readme/DESCRIPTION.rst

This module adds a new splitting mode to the `stock_split_picking` module.

> The new mode `Quantity of kits` allows to split a transfer by a specified number of kits.
> A new field on the wizard also allows specify a sort order for the stock moves before
> splitting the transfer.

Reviewer: ```suggestion
The new mode `Quantity of kits` allows splitting a transfer by a specified number of kits.
A new field in the wizard also allows specifying a sort order for the stock moves before
splitting the transfer.
```
